virttest/guest_agent.py

```diff
@@ -581,6 +581,14 @@ def set_time(self, nanoseconds=None):
         return self.cmd(cmd=cmd, args=args)
 
     @error_context.context_aware
+    def guest_info(self):
```

Comment:
the function 'guest_info' is newly added, later than '_get_supported_cmds', I will update it